sdk/python/tests/unit/cli/test_cli_chdir.py

```diff
@@ -18,14 +18,16 @@ def test_cli_chdir() -> None:
         repo_path = temp_path / "my_project" / "feature_repo"
         assert result.returncode == 0
 
-        result = runner.run(["--chdir", repo_path, "apply"], cwd=temp_path)
+        result = runner.run(["--chdir", str(repo_path), "apply"], cwd=temp_path)
```

Comment:
should we need casting?

Reply:
Yes, the type of the second argument of runner.run() is "str". The "repo_path" is an object.

Reply:
should we cast once for all from line 18 instead of repeated `str()` calls